.githooks/pre-push

```diff
@@ -1,3 +1,58 @@
 #!/bin/sh
+
+
+# git for-each-ref refs/tags  # see which tags are annotated and which are lightweight. Annotated tags are "tag" objects.
+# # Set these so your commits and tags are always signed
+# git config commit.gpgsign true
+# git config tag.gpgsign true
+
+verify_commit_signed() {
+    if git verify-commit HEAD &> /dev/null; then
+        :
+        # echo "HEAD commit seems signed..."
```

Comment:
do you want to remove the comments

Reply:
It looks like a useful comment.

Reply:
my comments disappeared?
I had said I was on the fence and usually don't leave comments like this but I kept uncommenting while debugging so I decided to leave them in. :shrug: